packages/e2e/test/projection/single-tenant-utxo.test.ts

```diff
@@ -47,7 +47,8 @@ const createDataSource = () =>
       Postgres.BlockDataEntity,
       Postgres.AssetEntity,
       Postgres.TokensEntity,
-      Postgres.OutputEntity
+      Postgres.OutputEntity,
+      Postgres.NftMetadataEntity
```

Comment:
This is the cost of having `AssetEntity.nftMetadata` column: all relations require you to explicitly specify the entity. Perhaps there is some workaround we could implement, but I think for now it's fine.

Reply:
Can we resolve this conversation @mkazlauskas @greatertomi ?